src/qtt/__init__.py

```diff
@@ -170,7 +170,7 @@ def _copy_to_str(x, memo=None):
 
 
 def _setstate(self, d):
-    self.name = d
+    self._short_name = d
```

Comment:
This does change the functionality, but I think it is ok since I doubt that we are using the serialization of Parameters with pickle.